### PR TITLE
New package: Nostalgia-09.AeroChat version 1.0.1-rc.3

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132860)

### CODE REVIEW

File: manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Nostalgia-09.AeroChat
+PackageVersion: 1.0.1-rc.3
+PackageLocale: en-US
+Publisher: Nostalgia-09
+PublisherUrl: https://discord.gg/3XfKgDMFBS
+PublisherSupportUrl: https://github.com/Nostalgia-09/AeroChat/issues
+PackageName: AeroChat
```
Comment (Trenly) [inline manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml:10]:
```suggestion
PackageName: Windows Live Messenger
```
Should match control panel

File: manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Nostalgia-09.AeroChat
+PackageVersion: 1.0.1-rc.3
+PackageLocale: en-US
+Publisher: Nostalgia-09
```
Comment (Trenly) [inline manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml:7]:
```suggestion
Publisher: Madeline, with special thanks to luisl and the Nostalgia '09 team
```
Should match control panel

File: manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+PublisherSupportUrl: https://github.com/Nostalgia-09/AeroChat/issues
+PackageName: AeroChat
+PackageUrl: https://github.com/Nostalgia-09/AeroChat
+License: MIT
+LicenseUrl: https://github.com/Nostalgia-09/AeroChat/blob/main/LICENSE
+ShortDescription: A Discord client themed to look like WLM 09, in React and Typescript.
+ReleaseNotes: |-
+  - Cashapp
+  - Gift cards of most varieties (preferably Amazon but beggars can't be choosers)
+  - Any sort of Discord Nitro subscription
```
Comment (Trenly) [inline manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml:18]:
```suggestion
ReleaseNotes: |-
  As per vote in the Nostalgia '09 Discord server, this update adds a dynamic banner system (the things on the 
  bottom of the homepage). This means that now, I can accept monthly donations in order to pay for things like 
  the website, and in return, donators can have their banners in the project for the duration of a month. If you're
  interested in donating, in return for a banner image which navigates to a URL on click:

  Join the [Discord server](https://discord.gg/nP9SxVQGnu)
  Tag me (@/notnullptr) and say that you want to donate!
  I'll DM you with some info and stuff and how to donate.
  
  Payment methods accepted:
  - Cashapp
  - Gift cards of most varieties (preferably Amazon but beggars can't be choosers)
  - Any sort of Discord Nitro subscription
  
  These will be toggle-able in a future update, but I'd like to push this change first.
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by Exorcism0666** [comment]
@Trenly help 😅⬆

**Comment by Trenly** [comment]
> @Trenly help 😅⬆

You'll have to try and find the silent install switches. I don’t know what they are

**Comment by Exorcism0666** [comment]
@Trenly So I asked the author for silent installation, I put in the commands, I don't know what's wrong again 💀

**Comment by Trenly** [comment]
> @Trenly So I asked the author for silent installation, I put the commands, I don't know what's wrong again 💀

You'll have to add it for both `Silent` and `SilentWithProgress` since winget defaults to `SilentWithProgress` switches unless the `--silent` parameter is used

**Comment by Exorcism0666** [comment]
@Trenly I'm really sorry for the number of notifications each time, it still doesn't work 💀

**Comment by Exorcism0666** [comment]
image
@OfficialEsco @Trenly Installation work, so idk what's wrong

File: manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+PackageIdentifier: Nostalgia-09.AeroChat
+PackageVersion: 1.0.1-rc.3
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2024-01-04
+InstallerSwitches:
+  Silent: /S --force-run
+  SilentWithProgress: /S --force-run
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.installer.yaml:13]:
```suggestion
- Architecture: x64
```
From validation pipeline :D
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20240107174905

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
For future troubleshooting
1. Click on the Checks tab
2. Expand "Azure Pipelines"
3. Select "WinGetSvc-Validation"
4. Select "0 errors / 3 warnings"
5. Select "4 published; 1 consumed"
6. Hover over and click on the 3 dots for "InstallationVerificationLogs"
7. Select "Download artifacts"
8. Browse the files, sometimes there are screenshots of the error, other times there are vague error codes

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**